config/zones/BE.yaml

```
@@ -15,6 +15,8 @@ capacity:
   oil: 455
   solar: 8595
   wind: 5315
+  battery: 50
+  battery storage: 100
```

```suggestion
  battery storage: 50
```
We don't support the storage values and only the discharge value should be added, at least for now.

We also need to add the source to `DATA_SOURCES.md` before we can merge this.